csharp-design-patterns/AggregateDesign/AggregateDesign/DataAccess/OrdersDbContext.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Microsoft.EntityFrameworkCore;
+
+namespace AggregateDesign.DataAccess;
+
+#nullable disable
+
+public class OrdersDbContext : DbContext
+{
+    public DbSet<OrderModel> orders { get; set; }
+    public DbSet<OrderItemModel> orderItems { get; set; }
```

Comment:
Properties should to be in UpperCamelCasing.

Reply:
done